Lib/fontbakery/checkrunner.py

```diff
@@ -1399,9 +1396,7 @@ def _load_spec_imports(self, symbol_table):
     if 'spec_imports' not in symbol_table:
       return results
 
-    # str for PY 3.x basetring for PY 2.x
-    is_string = lambda value: isinstance(value, \
-                        str if sys.version_info[0] == 3 else basestring) # NOQA
+    is_string = lambda value: isinstance(value, str) # NOQA
```

Comment:
Do we need this now? 🤔

Reply:
Hm, it's only used a few lines down. So, no, we can just use ` if isinstance(value, str):` below 😃

Reply:
Nope, removed!